README.md

```diff
@@ -0,0 +1,37 @@
+# Mattermost Suggestions Plugin
+[![CircleCI](https://circleci.com/gh/iomodo/mattermost-plugin-suggestions.svg?style=svg)](https://circleci.com/gh/iomodo/mattermost-plugin-suggestions)
```

Comment:
You still have here the `iomodo` url instead of the `mattermost` one. (And the same in the next line)